foundry.toml

```diff
@@ -1,46 +1,120 @@
+
+    # Test Configuration
 
-[fmt]
-bracket_spacing = false
-int_types = "long"
-line_length = 120
-multiline_func_header = "params_first"
-number_underscore = "thousands"
-quote_style = "double"
-tab_width = 4
+    # Verbosity level during test execution. Higher levels provide more detailed information:
+    # - 2 (-vv): Logs emitted during tests are displayed.
+    # - 3 (-vvv): Stack traces for failing tests are displayed.
+    # - 4 (-vvvv): Stack traces for all tests and setup traces for failing tests are displayed.
+    # - 5 (-vvvvv): Stack and setup traces are always displayed.
+    verbosity = 0
+    # Enables the Foreign Function Interface (FFI) cheatcode. 
+    # WARNING: This allows arbitrary programs to run on your computer, which poses security risks.
+    ffi = false
+    # Contracts to include in gas reports. By default, all contracts are included.
+    gas_reports = ["./src/contracts/**/*"]
+    # Show test execution progress if set to true.
+    show_progress = true
+    # Print compiled contract names if set to true.
+    names = true
+    # Print compiled contract sizes if set to true.
+    sizes = true
+    # Sparse mode only compiles files that match certain criteria.
+    sparse_mode = true
+    
+    no_match_test = "queueUpgrade"
+    no_match_path = "script/releases/**/*.sol"
+    fs_permissions = [{ access = "read-write", path = "./"}]
 
-# See more config options https://github.com/gakonst/foundry/tree/master/config
+[profile.default.fmt]
+    # Single-line vs multi-line statement blocks
+    single_line_statement_blocks = "preserve"  # Options: "single", "multi", "preserve"
+    # Formatting style for long function headers
+    multiline_func_header = "params_first"  # Options: "attributes_first", "params_first", "all"
+    # Sort import statements alphabetically
+    sort_imports = true  
+    # Add empty lines before and after contract definitions
+    contract_new_lines = false  
+    # Space between 'override' keyword and parent contracts
+    override_spacing = true  
+    # Maximum line length where formatter will wrap the line
+    line_length = 120  # Default: 120
+    # Number of spaces per indentation level
+    tab_width = 4  # Default: 4
+    # Whether to print spaces between brackets
+    bracket_spacing = false  
+    # Style of uint/int256 types
+    int_types = "long"  # Options: "long", "short", "preserve"
+    # Quotation mark style
+    quote_style = "double"  # Options: "double", "single", "preserve"
+    # Style of underscores in number literals
+    number_underscore = "thousands"  # Options: "preserve", "thousands", "remove"
+    # Whether or not to wrap comments at line_length
+    wrap_comments = false  
+    # List of files to ignore during formatting (can use glob patterns)
+    ignore = [
+        "./script/**/*",
+        "./src/test/**/*"
+    ]
+
+[profile.test.fmt]
+    int_types = "short"
+    line_length = 140
+    ignore = [
+        "./src/contracts/**/*"
+    ]
```

Comment:
Throughout out tests, we typically opt for short uint/int types, and prefer longer line length.